GS_example.py

```diff
@@ -36,21 +40,27 @@
           np.arange(-params.matrix_size / 2, params.matrix_size / 2) * params.pixel] for y in
          np.arange(-params.matrix_size / 2, params.matrix_size / 2) * params.pixel])
 
+    # Prepare optimizer
     GS = GerchbergSaxton(params)
 
+    # Run optimizer
     res = GS.optimize(LightField(amp, phase), LightField(target, phase), 5)
```

Comment:
What is `5`? I assume number of iterations. Keyword - _assume_. I'd make it more descriptive, its supposed to be a demo for user who most probably does not know the API. I'd add `5` as named param

Reply:
I agree, `iterations=5` would look better

Reply:
Good idea, added to both optimization examples.